test/e2e/infra/vagrant/playbook/roles/control-plane/templates/kubeadm.conf.j2

```diff
@@ -13,3 +13,5 @@ networking:
 apiServer:
   certSANs:
   - "{{ k8s_api_server_ip }}"
+  extraArgs:
+    feature-gates: "NetworkPolicyEndPort=true"
```

Comment:
you are enabling EndPort in the Vagrant testbed, which is fine by me, but there is no e2e test for EndPort yet?

Reply:
I believe @GraysonWu mentioned that he is looking into adding the feature gate in `antrea/ci/jenkins/` and `antrea/ci/kind/`, maybe you could share a bit more info on that front? Also, some amount of documentation on how the feature gate can be enabled for the k8s-apiserver will be very useful.

Reply:
Just added e2e test and enabled endport feature gate in Kind. 
About Jenkins, I asked zhecheng for more information. Let's see his reply if they support 1.21.

Reply:
As @lzhecheng said, the testbed hasn't upgraded to 1.21. He will add that feature gate while upgrading.